src/Core.UnitTests/LanguageTests.cs

```diff
@@ -67,6 +67,9 @@ public void Language_IsSupported_SupportedLanguage_IsTrue()
             Language.VBNET.IsSupported.Should().BeTrue();
             Language.Cpp.IsSupported.Should().BeTrue();
             Language.C.IsSupported.Should().BeTrue();
```

Comment:
I felt weird only adding the secret tests so I added the js and ts stuff aswell.

Reply:
Thanks! If you spot a test hole when changing a component, it's a practice to add some tests if it doesn't take too much effort.